UPGRADING.md

```diff
@@ -9,6 +54,10 @@ The previous Bugsnag React Native SDK – [`bugsnag-react-native`](https://githu
 
 See the [React Native upgrade guide](/packages/react-native/UPGRADING.md) for specific instructions on how to upgrade from `bugsnag-react-native` to `@bugsnag/react-native`.
 
+#### plugin-contextualize
+
+Unhandled errors that occur within a contextualize context now respect the `autoDetectErrors`` and `enabledErrorTypes`` configuration options. Previously unhandled errors would have been caught regardless of the configuration.
```

Comment:
Is the change also that we previously _handled_ the error, so the thread didn't crash?

Reply:
That also applies to unhandled errors in requests so will need mentioning elsewhere as well

Reply:
I've discussed this in two places - what do you think to the wording?